test/known_issues/test-vm-data-property-writable.js

```diff
@@ -0,0 +1,17 @@
+'use strict';
+// Refs: https://github.com/nodejs/node/issues/10223
+
+require('../common');
+const vm = require('vm');
+const assert = require('assert');
+
+const context = vm.createContext({});
+
+const code = `
+   Object.defineProperty(this, 'foo', {value: 5});
+   Object.getOwnPropertyDescriptor(this, 'foo').writable;
```

Comment:
I'd return the descriptor and then check that .writable, .enumerable and .configurable are all false.